SNIPS/snip-75.md

```diff
@@ -65,6 +65,24 @@ This is an emergency flow, initiated by the security council; therefore, it is i
 
 **Note**: Regular time delay SHOULD be set to at least 7 days to be in line with l2beat guidelines.
 
+### Pausing Flow
+
+Each pausing flow will include two functions:
+
+- **Pause**: Stopping the affected smart contract from further changes of state.
+- **Unpause**: Resuming normal operation.
+
+#### Flow 1: Pausing the Contract
+
+In case of critical incidents, a monitoring entity has the ability to pause the affected contracts immediately. This pause mechanism serves as a first line of defense to contain potential losses while the Security Council deliberates on a potential upgrade.
```

Comment:
Does the pause inherently have an expiration?
Like pause until unpause or at most for 1 day

Reply:
There is no automatic unpause in the current design